### PR TITLE
feat: User pages integrate API

### PR DESCRIPTION
**Pull request by whDongRui**
* fix: App query and delete according to status

### CODE REVIEW

File: src/pages/Admin/Users/GroupCard/index.jsx [hunk 1]
```diff
@@ -1,34 +1,35 @@
 import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
+import { translate } from 'react-i18next';
 import classnames from 'classnames';
 
 import styles from './index.scss';
 
+@translate()
 export default class GroupCard extends PureComponent {
   static propTypes = {
```
Comment (iwisunny) [inline src/pages/Admin/Users/GroupCard/index.jsx]: Should also add `defaultProps`

File: src/pages/Admin/Users/GroupCard/index.jsx [hunk 2]
```diff
@@ -1,34 +1,35 @@
...
+        {groups.map((data, index) => (
+          <li
+            key={data.id || index}
+            onClick={() => {
+              selectCard(data);
+            }}
+            className={classnames({ [styles.current]: data.value === selectValue })}
+          >
+            <div className={styles.name}>{t(data.name)}</div>
+            {data.id && <div className={styles.id}>id:{data.id}</div>}
```
Comment (iwisunny) [inline src/pages/Admin/Users/GroupCard/index.jsx]: what if `data.id` is 0 ?

File: src/pages/Admin/Users/index.jsx [hunk 3]
```diff
@@ -191,39 +222,103 @@ export default class Users extends Component {
...
-            <label>{t('Name')}</label>
-            <Input name="name" autoFocus maxLength="50" />
-          </div>
+        <form className="formContent">
+          {userDetail.user_id && (
+            <div className="inputItem">
+              <label>{t('Name')}</label>
+              <Input
+                name="name"
+                maxLength="50"
```
Comment (iwisunny) [inline src/pages/Admin/Users/index.jsx]: `maxLength` should be number ?

File: src/pages/Admin/Users/index.jsx [hunk 4]
```diff
@@ -191,39 +222,103 @@ export default class Users extends Component {
...
-            <Input name="name" autoFocus maxLength="50" />
+            <Input
+              name="email"
+              maxLength="50"
+              placeholer="username@example.com"
+              value={userDetail.email}
+              onChange={e => {
+                changeUser(e, 'email');
+              }}
+              pattern="^\w+([-+.]\w+)*@\w+([-.]\w+)*\.\w+([-.]\w+)*$"
```
Comment (iwisunny) [inline src/pages/Admin/Users/index.jsx]: Email must prefix  with regexp `\w`? 
Checkout: https://www.regextester.com/97767

File: src/pages/Admin/Users/index.jsx [hunk 5]
```diff
@@ -191,39 +222,103 @@ export default class Users extends Component {
...
+              <Select.Option value="global_admin">{t('Administrator')}</Select.Option>
+            </Select>
           </div>
-          <div className="inputItem textareaItem">
+          <div className="inputItem">
+            <label>{t('Password')}</label>
+            <Input
+              name="password"
+              type="password"
+              maxLength="50"
```
Comment (iwisunny) [inline src/pages/Admin/Users/index.jsx]: `maxLength` should be number

File: src/pages/Admin/Users/index.jsx [hunk 6]
```diff
@@ -248,27 +343,28 @@ export default class Users extends Component {
...
       selectValue,
-      selectItem,
+      selectGroupId,
+      selectRoleId,
       selectName
     } = userStore;
     const groups = userStore.groups.toJSON();
-    //const roles = userStore.roles.toJSON();
+
     const roles = [
```
Comment (iwisunny) [inline src/pages/Admin/Users/index.jsx]: mock data moved to external file is better

File: src/pages/Home/index.jsx [hunk 7]
```diff
@@ -19,16 +19,16 @@ import styles from './index.scss';
 @observer
 export default class Home extends Component {
   static async onEnter({ categoryStore, appStore }, { category, search }) {
-    const params = {};
+    appStore.loadPageInit();
+    await categoryStore.fetchAll();
+
+    const params = { status: 'active' };
```
Comment (iwisunny) [inline src/pages/Home/index.jsx]: define variable at the begining of function is better

File: src/pages/Admin/Users/GroupCard/index.jsx (hunk 1 above)
Comment (whDongRui) [reply src/pages/Admin/Users/GroupCard/index.jsx]: ok.

File: src/pages/Admin/Users/GroupCard/index.jsx (hunk 2 above)
Comment (whDongRui) [reply src/pages/Admin/Users/GroupCard/index.jsx]: Add jude becase roles data from  front end,  groups data from api，id is string.

File: src/pages/Admin/Users/index.jsx (hunk 3 above)
Comment (whDongRui) [reply src/pages/Admin/Users/index.jsx]: ok.maxLength=50 has error note,  maxLength= {50}

File: src/pages/Admin/Users/index.jsx (hunk 5 above)
Comment (whDongRui) [reply src/pages/Admin/Users/index.jsx]: ok

File: src/pages/Admin/Users/index.jsx (hunk 6 above)
Comment (whDongRui) [reply src/pages/Admin/Users/index.jsx]: ok, add static file store mock data.

File: src/pages/Home/index.jsx (hunk 7 above)
Comment (whDongRui) [reply src/pages/Home/index.jsx]: ok, params is used below method.

File: src/pages/Admin/Users/index.jsx (hunk 4 above)
Comment (whDongRui) [reply src/pages/Admin/Users/index.jsx]: ok

File: src/pages/Admin/Users/index.jsx (hunk 3 above)
Comment (iwisunny) [reply src/pages/Admin/Users/index.jsx]: right, expression placed in {}

File: src/pages/Admin/Users/GroupCard/index.jsx (hunk 2 above)
Comment (iwisunny) [reply src/pages/Admin/Users/GroupCard/index.jsx]: ok

File: src/pages/Home/index.jsx (hunk 7 above)
Comment (iwisunny) [reply src/pages/Home/index.jsx]: yes, see javascript variable hoist

**Pull request merged**